packages/expo-yarn-workspaces/index.js

```diff
@@ -84,10 +67,7 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
       providesModuleNodeModules: [],
 
       // Ignore JS files in the native Android and Xcode projects
-      blacklistRE: exclusionList([
-        /.*\/android\/React(Android|Common)\/.*/,
-        /.*\/versioned-react-native\/.*/,
-      ]),
+      blockList: [/.*\/android\/React(Android|Common)\/.*/, /.*\/versioned-react-native\/.*/],
```

Comment:
`resolver.blockList` defaults to using `[/\/__tests__\/.*/]`: https://github.com/facebook/metro/blob/76a4e976d589c02d7d1c03e3e4c6b3ead22801f4/packages/metro-config/src/defaults/exclusionList.js#L14

It seems like a good idea to continue ignoring test files -- should we keep this?